casepro/backend/__init__.py

```diff
@@ -199,6 +199,15 @@ def fetch_contact_messages(self, org, contact, created_after, created_before):
                  Message.as_json() for incoming messages and Outgoing.as_json() for outgoing messages.
         """
 
+    @abstractmethod
+    def get_url_patterns(self):
```

Comment:
Wondering where we're going with this? Is the actual pre-approved responses stuff in another app?